### DISCUSSION TITLE
Auctions + Craft

### DISCUSSION THREAD

**Discussion by LucasAG95** [sunflower-land/sunflower-land#4761] [Ideas]
With the arrival of animals, a lot of new resources were introduced, and at first, they don't seem very useful, which is normal because they are new.

You probably have something planned, but I would like to share an idea I had related to Crafting Box + Auctions, and Stella too!

The idea would be to transform today's auctions into 'Project' auctions, where instead of buying the NFT at the auction, the person would buy the project to craft the NFT, thus using both new and old resources in the Crafting Box, and the same with the items in Stella.

At first, I believe many wouldn't like the idea because it would be more complex, and they might think there will be a lot of expenses besides the auction cost. But I don't think so, because knowing this, maybe they would bid less at auctions to buy or make the necessary resources to craft the piece. This would create a demand for more resources and new planning for the season.

Another suggestion would be that SFL and GEMs auctions give the complete item, and the seasonal race would give the project to craft the NFTs, in a world where seasonal races last 5 weeks before the auctions, just like this one.

I'm not sure if I expressed the idea as well as it is in my head, but even if it's not implemented, I hope it helps with other ideas!!
![Captura de tela 2024-11-12 194824](https://github.com/user-attachments/assets/3f304f3a-9fc1-4405-afb9-773898e68a7f)

**Comment by TexugoJoga** [comment]
I really liked the idea! This would also make it easier to mint the item, as you could pay the gas to generate the item only when you were going to create it in the crafting box, eliminating the need to do this as soon as the item's auction ends.

**Comment by Xeko-92** [comment]
The problem with this idea is that the crafting box isn't limited to 1 time crafts. So not only would you be changing how obtaining items works from Stella & the auction, you would also be changing how the crafting box works.

Also how would this work for minting the item? Do you need to mint it when you win it as well as when you're crafting it?

**Reply by TexugoJoga** [reply]
The item you will receive at auction is a single-use blueprint, so you could only craft it once. And my idea is that you receive this recipe and you don't need to mint the recipe as an nft at the beginning, the recipe would become an nft as soon as you use it to actually create the item.

**Reply by Xeko-92** [reply]
To me it just seems like extra dev work for not much of a benefit. There are changes that this would require that would take time away from other things they're doing. They would also need to come up with recipes for each of these items every season.

**Reply by TexugoJoga** [reply]
I see an advantage in this because it gives more possibilities of uses for the items that animals drop. This would certainly impact the desire to have NFTs for the future and perhaps make the prices of the items more realistic with the cost of producing them.

**Reply by Xeko-92** [reply]
Well they should be adding items that make the animal drops worth it, not insert them into something where we didn't need them before imo. Also, the animal drops will only be used for clothing really, we'll need much more than animal drops for most NFTs.

Like I said, it feels like it just complicates something that doesn't need complication.
